plugins/json-multisheet-extractor/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@flatfile/plugin-json-multisheet-extractor",
+  "version": "0.1.0",
+  "url": "https://github.com/FlatFilers/flatfile-plugins/tree/main/plugins/json-multisheet-extractor",
+  "description": "A plugin for parsing json files into multiple sheets in Flatfile.",
```

Comment:
`json` to `JSON` https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1825998708